### PR TITLE
assignment_operator.cpp update

### PR DESCRIPTION
**Pull request by SamLokh**

### CODE REVIEW

**Review by ayaankhan98 — COMMENTED** [review]
tests not written

**Comment by SamLokh** [comment]
> tests not written

I'll add the tests.

**Comment by ayaankhan98** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
- Added 1
           
```


See the complete overview on [Codacy](https://app.codacy.com/gh/ayaankhan98/bigINT/pullRequest?prid=6287175&bid=20456491)

File: tests/operators/assignmentoperator_test.cpp [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+    bigIntVarA = 1928374;
+    assert(bigIntVarA == std::string("1928374"));
+
+    //Assigning one largeInt variable to another largeInt variable
+    bigIntVarB = bigIntVarA;
+    assert(bigIntVarB == std::string("1928374"));
+
+
+    //Assigning int variable to largeInt variable
+    bigIntVarC = intVarA;
```
Comment (ayaankhan98) [inline tests/operators/assignmentoperator_test.cpp:23]: Codacy found an issue: [Variable 'bigIntVarC' is assigned a value that is never used.](https://app.codacy.com/gh/ayaankhan98/bigINT/pullRequest?prid=6287175)

**Comment by ayaankhan98** [comment]
Tests are failing
```
Test project /home/ayaan/workspace/bigINT/build/tests/operators
    Start 1: assignmentoperator_test
1/1 Test #1: assignmentoperator_test ..........Child aborted***Exception:   0.15 sec

0% tests passed, 1 tests failed out of 1

Total Test time (real) =   0.15 sec

The following tests FAILED:
	  1 - assignmentoperator_test (Child aborted)
Errors while running CTest
```

**Pull request closed without merging**